jwql/website/apps/jwql/migrations/0018_centralstorage_fgsanomaly_filesystemcharacteristics_and_more.py

```diff
@@ -0,0 +1,612 @@
+# Generated by Django 4.2.5 on 2024-02-20 20:19
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Nothing at all. Same with 19 and 20. For whatever reason, things went as follows:

- I create a migration with makemigration
- I run it with migrate.py and it causes an error or doesn't actually implement the changes, and it suggests running makemigration again
- I do so, and it makes a second pretty much identical migration
- This one works

I have, frankly, no idea what happened.

Reply:
Strange, maybe when @BradleySappington does his deep dive he might see something we aren't seeing.

Reply:
I'm suggesting some overall structural changes, and if agreed upon and implemented we should put the database back to where you began, and re-run make migrations.  If we run into a weird situation like this lets address it when it pops up.  I noticed (from experience) that if something is weird locally, it will not be smooth sailing down the line.

Reply:
looks like the migrations got worked out with those changes